Makefile

```
@@ -1,3 +1,7 @@
 lint:
 	shellcheck src/*.sh
```

shellcheck also supports .bats files as well.

```suggestion
	shellcheck src/*.sh test/*.bats
```